theano/gpuarray/tests/test_pickle.py

```diff
@@ -29,7 +29,7 @@
         cuda_ndarray.use('gpu', default_to_move_computation_to_gpu=False,
                          move_shared_float32_to_gpu=False,
                          enable_cuda=False)
-        theano.sandbox.gpuarray.init_dev('cuda')
```

Comment:
I mean that this should use either the normal code to get the test context or just initiailize straight away and not go through cuda_ndarray (which doesn't work anymore).

I remember that this was done this way because if you import config it raises SkipTest when pygpu is not available.  I think that catching SkipTest and noting that is a better way than what is here.